google/cloud/spanner_dbapi/client_side_statement_executor.py

```diff
@@ -63,24 +64,26 @@ def execute(cursor: "Cursor", parsed_statement: ParsedStatement):
         connection.rollback()
         return None
     if statement_type == ClientSideStatementType.SHOW_COMMIT_TIMESTAMP:
-        if connection._transaction is None:
-            committed_timestamp = None
-        else:
-            committed_timestamp = connection._transaction.committed
+        if (
+            connection._transaction is not None
+            and connection._transaction.committed is not None
+        ):
+            column_values.append(connection._transaction.committed)
```

Comment:
This seems to change the behavior of `show commit_timestamp`:
1. Previously (before this change), if the transaction had not committed, then `show commit_timestamp` would return one row with value `None`.
2. Now `show commit_timestamp` will return a result set with zero row.

Both return types would be OK, but we should not change the behavior unintentionally, and also not once one of the two behaviors has been included in a public release. Are you sure that 'no rows' is the return value that you would prefer in those cases? (I assume that we have not done a release of the dbapi driver since support for `show commit_timestamp` was added. If we have, then we need to undo this change in this PR.)

Reply:
So the release has not happened in last couple of months.

Also I think there is no change in the resultset from before as we would have returned a result set with zero row previously as well as we were checking that `column_value is not None` on line 100